Tools/serial/rc.serial_port.jinja

```diff
@@ -5,7 +5,7 @@
 set SERIAL_DEV none
 {% for serial_device in serial_devices -%}
 if param compare "$PRT" {{ serial_device.index }}; then
-	if [ "x$PRT_{{ serial_device.tag }}_" = "x" ]; then
+	if [ "$PRT_{{ serial_device.tag }}_" = "x" ]; then
```

Comment:
Did you check if this works with nuttx?
It needs to be:
```suggestion
	if [ "$PRT_{{ serial_device.tag }}_" = "" ]; then
```

Or:
```suggestion
	if [ -z "$PRT_{{ serial_device.tag }}_" ]; then
```

Reply:
I haven't checked if this works with NuttX, can you help me with a few steps on how to bench test this?

Reply:
I checked the first one, it works.
You can use this on the shell:
```
if [ "$a" = "" ]; then; echo empty; else; echo nonempty; fi
set a xy
if [ "$a" = "" ]; then; echo empty; else; echo nonempty; fi
```